docs/file-reference.md

```diff
@@ -455,4 +505,8 @@ volumeClaims:
 configMaps:
 - data:
     MYSQL_DATABASE: wordpress
+secrets:
+- name: wordpress
+  data:
+    MYSQL_ROOT_PASSWORD: YWRtaW4=
```

Comment:
where did you got web.yaml?

only db is defined there no?


To be honest, that whole example doesn't make sense :-)